arm9/modules/48/asm/module_48.s

```diff
@@ -953,14 +953,31 @@ _02254F5C: .word 0x000003E7
 	thumb_func_end MOD48_02254EF4
 
 	.section .rodata
-	; 0x02254F60
-	.byte 0x3E, 0x62, 0x58, 0x88, 0x4E, 0x72, 0x98, 0xC8, 0x56, 0x7A, 0x18, 0x48, 0xB1, 0x49, 0x25, 0x02
-	.byte 0xF1, 0x49, 0x25, 0x02, 0x6D, 0x4A, 0x25, 0x02, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00
+
+	.global MOD48_02254F60
+MOD48_02254F60: ; 0x02254F60
```

Comment:
Arrays passed to the first arg of MOD20_02254130 appear to be Nx4 u8 arrays. Arrange accordingly.